manifests/issue.pp

```diff
@@ -0,0 +1,34 @@
+# === Copyright
+#
+# Copyright 2014, Deutsche Telekom AG
```

Comment:
@artem-sidorenko Apologies, Will change it. As I have recently ( a month ago ) started working on puppet, I was just learning and because of that.. Just copied it. Could you please let me know what should be the copyright here?